app/controllers/staff/planned_disbursements_controller.rb

```diff
@@ -15,7 +15,16 @@ def create
     authorize @activity
 
     history = history_for_create
-    history.set_value(planned_disbursement_params[:value])
+
+    begin
+      history.set_value(planned_disbursement_params[:value])
+    rescue PlannedDisbursementHistory::SequenceError => _e
+      @planned_disbursement = PlannedDisbursement.new(planned_disbursement_params)
+      @planned_disbursement.parent_activity = @activity
+      @planned_disbursement.errors.add(:financial_quarter, t("activerecord.errors.models.planned_disbursement.attributes.financial_quarter.in_the_past"))
```

Comment:
Might as well use `errors.add` as Rails intended, and let it do the I18n lookup  here:

```ruby
@planned_disbursement.errors.add(:financial_quarter, :in_the_past)
```

Reply:
Symbol, not string! 🤦🏻‍♀️  (I had tried that before, but with `"in_the_past"`.) Thank you!